tests/apply.rs

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+pub mod util;
+
+use assert_cmd::prelude::*; // Add methods on commands
+use predicates::prelude::*; // Used for writing assertions
+
+#[tokio::test]
+#[should_panic(expected = "not yet implemented")]
+async fn test_apply() {
+    let mut c = util::setup().await.expect("faild to setup");
+    let apply_cmd = c.args(&["--region", "local", "admin", "apply"]);
+    apply_cmd
+        .assert()
+        .failure()
+        .stderr(predicate::str::contains("not yet implemented"));
+
+    let _apply_cmd_with_long = c.args(&["--region", "local", "admin", "apply", "--test-apply"]);
```

Comment:
I believe this output should be inspected. Command execution must be failed with panic.